hoomd/hpmc/PatchEnergyJIT.cc

```diff
@@ -69,7 +69,6 @@ void export_PatchEnergyJIT(pybind11::module& m)
                             Scalar,
                             pybind11::array_t<float>>())
         .def_property("r_cut", &PatchEnergyJIT::getRCut, &PatchEnergyJIT::setRCut)
-        .def("energy", &PatchEnergyJIT::energy)
```

Comment:
Why remove this?

Reply:
not sure. added it back in and tests pass fine and it builds.